src/common/services/session/session.service.js

```diff
@@ -276,19 +276,19 @@ const session = /* @ngInject */ function ($cookies, $rootScope, $http, $timeout,
     return envService.read('oktaUrl')
   }
 
-  function removeLocationOnLogin () {
-    $window.sessionStorage.removeItem(locationOnLogin)
+  function removeStoredLocation () {
```

Comment:
Thanks for renaming these! What do you think about `getRedirectLocation` and `clearRedirectLocation`? I that communicates a little better the intent of what the stored location is used for.